src/util.js

```diff
@@ -1,3 +1,13 @@
+export let BLANKS = [undefined, null, false];
```

Comment:
Why export this if it's encapsulated in `isBlank`? (FWIW, I have performance anxiety, same as with `isVoidElement`.)